tardis/model/base.py

```diff
@@ -0,0 +1,7 @@
+class Radial1DModel(object):
+    def __init__(self, plasma, atom_data, structure):
+        pass
+
+    @classmethod
+    def from_config(self, config):
```

Comment:
@wkerzendorf What do you think about: `__init__(self, plasma, atom_data, velocities, densities, structure, abundances)`

`t_inner` could be read from `plasma.t_inner`,
`v_inner` and `v_outer` could be read from `structure` which could be a new object.

Does `velocities` have anything to do with `v_inner` or `v_outer`?